### PR TITLE
docs: update PR template for GitHub

### PR DESCRIPTION
**Pull request by snapsnapturtle**
**What:**
Add explanation on what to do with visual changes: add new design-review label.​

**Why:**
To allow designers to review changes that they are able to properly review. It helps them prioritise their work without having to look through all technical PRs.
​

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ merge of your pull request!
 -->
 
 **What:**
-<!-- Declarative and short sentence of what this PR accomplish -->
+<!-- Declarative and short sentence of what this PR accomplish. If the PR contains visual changes, please add the design-review label to the PR -->
```
Comment (martimalek) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: I'm not a native english speaker but I believe the word `accomplish` here should be `accomplishes`. I know this is not a mistake in the new text but maybe we can use this PR to change it?

Comment (snapsnapturtle) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Absolutely right, I'm also not a native speaker, but accomplishes is correct.

Comment (snapsnapturtle) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
<!-- Declarative and short sentence of what this PR accomplishes. If the PR contains visual changes, please add the design-review label to the PR -->
```

**Review by martimalek — APPROVED** [review]

**Review by rafael-sepeda — APPROVED** [review]
Thanks!

**Pull request merged**